docs/source/user_guide/runtime-conf.md

```diff
@@ -35,7 +35,7 @@ elyra-metadata install runtimes --schema_name=kfp \
        --name=my_kfp \
        --display_name="My Kubeflow Pipeline Runtime" \
        --api_endpoint=https://kubernetes-service.ibm.com/pipeline \
```

Comment:
Looking into [recipes/deploying-kubeflow-locally-for-dev.md](https://elyra.readthedocs.io/en/latest/recipes/deploying-kubeflow-locally-for-dev.html), I was wondering if we should use the same pattern here as if you were deploying it locally and use `--api_endpoint=http://localhost:31380/pipeline`

Reply:
That makes sense to me, but I can't say if localhost:31380 actually works (I have not run through the other examples - just using an existing kfp).  It does sound a lot more likely to work in an example than `kubernetes-service.ibm.com` though